.github/workflows/vm-netbsd.yml

```diff
@@ -10,34 +10,35 @@ on:
 
 jobs:
   build-netbsd:
-    if: false && (github.event_name == 'push' || github.event.pull_request.merged == true)
-    name: NetBSD ${{ matrix.osversion }} ${{ matrix.abi == '64' && ' ' || matrix.abi }} ${{ matrix.nroff == 'true' && 'no-roff' || ' ' }} ${{ matrix.cflags == '' && ' ' || 'no-obsolete' }}
+    if: true && (github.event_name == 'push' || github.event.pull_request.merged == true)
+    name: >
+      NetBSD
+      ${{ matrix.osversion }}
+      ${{ matrix.abi == '64' && ' ' || matrix.abi }}
+      ${{ matrix.nroff == 'true' && 'no-roff' || ' ' }}
+      ${{ matrix.cflags == '' && ' ' || 'no-obsolete' }}
     runs-on: ubuntu-latest
-    env:
-      CFLAGS: ${{ matrix.cflags }}
-      NQABI: ${{ matrix.abi }}
-      NROFF: ${{ matrix.nroff }}
     strategy:
       fail-fast: false
       matrix:
         osversion: ["9.3"]
-        abi: [64] # 32
+        abi: [64, 32]
```

Comment:
This means you are trying 32-bit binaries on a 64-bit architecture? If so, quite extensive tests!

Reply:
Not currently working (because 32-bit tests won't link with 64-bit `libcheck`) and seems not worth the trouble, so I'm gonna remove it. I _would_ like a variety of NetBSD hardware architectures, as we now have with Alpine, but am not aware of any Actions designed for that. (My [nbvm](https://github.com/schmonz/nbvm) project may help someday but is nowhere near ready.)